processing-service-core/api/models/db/tables/tenantConfig.js

```diff
@@ -0,0 +1,29 @@
+// defines the structure representing the tenantConfig table in the database
+const Sequelize = require("sequelize");
+
+var tableDefinition = {
+    tenantId: {
+        allowNull: false,
+        field: "tenant_id",
+        primaryKey: true,
+        type: Sequelize.STRING(40)
+    },
+    key: {
+        allowNull: false,
+        field: "key",
+        primaryKey: true,
+        // TODO how to represent the foreign key constraint?
```

Comment:
Is there anything more to do here?

Reply:
Update to add in reference to the global config 'key' column. Couldn't find an example of doing so without creating an instance of the other model but source code indicates that table name can be used here.
http://docs.sequelizejs.com/manual/tutorial/models-definition.html#configuration
- describes setting up the association

The foreign key constraint was already being enforced as it is set up by the DB installer. This is just about having it written down in the definition here.